src/routes/settings/assistants/[assistantId]/+page.server.ts

```diff
@@ -88,18 +89,22 @@ export const actions: Actions = {
 			const prefixUrl = PUBLIC_SHARE_PREFIX || `${PUBLIC_ORIGIN || url.origin}${base}`;
 			const assistantUrl = `${prefixUrl}/assistant/${params.assistantId}`;
 
-			const assistant = await collections.assistants.findOne(
+			const assistant = await collections.assistants.findOne<Pick<Assistant, "name">>(
 				{ _id: new ObjectId(params.assistantId) },
 				{ projection: { name: 1 } }
 			);
 
+			const username = locals.user?.username;
+
 			const res = await fetch(WEBHOOK_URL_REPORT_ASSISTANT, {
 				method: "POST",
 				headers: {
 					"Content-type": "application/json",
 				},
 				body: JSON.stringify({
-					text: `Assistant <${assistantUrl}|${assistant?.name}> reported by <http://hf.co/${locals.user?.username}|${locals.user?.username}>. The following reason was given \n\n> ${result.data}`,
+					text: `Assistant <${assistantUrl}|${assistant?.name}> reported by ${
+						username ? `<http://hf.co/${username}|${username}>` : "non-logged in user"
+					}. The following reason was given \n\n> ${result.data}`,
```

Comment:
```suggestion
					}. \n\n> ${result.data}`,
```

shall we remove this ? Feels a bit unnecessary when looking at the slack formatting